converter/script.js

```diff
@@ -0,0 +1,33 @@
+function convert() {
+    function toFahrenheit(input) {
+        var celcius = ((parseInt(input, 10) - 32) *  (5 / 9)).toFixed(2) + "&#8451";
+        return celcius;
+    }
+
+    function toCelcius(input) {
+        var farenheit = ((parseInt(input, 10) * 1.8) + 32).toFixed(2) + "&#8457";
+        return farenheit;
+    }
+
+    function isFahrenheit(input) {
+        if (input[input.length - 1] === "f" ||  input[input.length - 1] === "F") {
+            return true;
+        }
```

Comment:
this could be shortened to `return input[input.length - 1] === "f" ||  input[input.length - 1] === "F"`. Same with `isCelsius`